params/variants.toml

```diff
@@ -1546,20 +1552,19 @@ selectorAffix."a/turnABase" = ""
 selectorAffix."a/single"    = ""
 selectorAffix.scripta       = ""
 
-[prime.a.variants-buildup.stages.double-storey-hook.hook-serifed]
+[prime.a.variants-buildup.stages.double-storey-hook.corner-hooked]
```

Comment:
Please do not rename this. The shape here in `a` does not shape like `r` (which has right angle).